Wikipedia/Code/NavigationStateController.swift

```diff
@@ -21,6 +21,78 @@ final class NavigationStateController: NSObject {
     private typealias Presentation = ViewController.Presentation
     private typealias Info = ViewController.Info
 
+    /// Finds the topmost article from persisted NavigationState and pushes it onto navigationController
+    @objc func restoreLastArticle(for navigationController: UINavigationController, in moc: NSManagedObjectContext, with theme: Theme, completion: @escaping () -> Void) {
+        guard let tabBarController = navigationController.viewControllers.first as? UITabBarController else {
```

Comment:
I think this warning can be alleviated with:
```swift
guard navigationController.viewControllers.first is UITabBarController else { ...
```

(Also very cool to see the Xcode Cloud warnings propagating up to the code review layer here in GitHub!)

Reply:
‼️ That is very cool! This is fixed.